Jenkinsfile

```diff
@@ -115,15 +122,3 @@ void withMavenEnv(List envVars = [], def body) {
         body.call()
     }
 }
-
-void renameFiles(def files, String prefix) {
```

Comment:
Why did you remove this? IIUC it was added to work around JUnit result bad aggregation between parallel branches.

Reply:
Only because we had been publishing artifacts from both Linux and Windows branches. There was no point in that; the Windows build is there to run tests on Windows, but the artifacts should be (nearly) identical. Now the build is publishing artifacts only from the Linux branch.